torch/_inductor/compile_fx.py

```diff
@@ -670,6 +682,11 @@ def fx_codegen_and_compile(
             payload_fn=lambda: gm.print_readable(print_output=False),
         )
         optimus_scuba_log["inductor"] = counters["inductor"]
+        runtime_env: RuntimeEnvironment = RuntimeEnvironment()
```

Comment:
maybe move conditional import here followed by scuba logging in same if-block.

Reply:
Thanks for the comment. The model_type and global_rank will return None if no runtime environment is found. I would like to put all fields update in the same place, thus users can see what have been logged easily. What do you think?